src/components/events/CheckInModal/index.tsx

```diff
@@ -35,6 +70,15 @@ const CheckInModal = ({ open, event, onClose }: CheckInModalProps) => {
 
   return (
     <Modal open={open} onClose={onClose}>
+      <Confetti
+        key={`${refreshConfetti}`}
```

Comment:
should it use `event.uuid` instead? and does this still draw confetti while the modal is closed?